README.md

```diff
@@ -42,11 +42,13 @@ The `canary` (latest master) version is:
 
 ## Supported TypeScript Version
 
-We will always endeavor to support the latest stable version of TypeScript.
+We will always endeavor to support the latest stable version of TypeScript. Sometimes, but not always, changes in TypeScript will not require breaking changes in this project, and so we are able to support more than one version of TypeScript.
```

Comment:
```suggestion
We will always endeavour to support the latest stable version of TypeScript. Sometimes, but not always, changes in TypeScript will not require breaking changes in this project, and so we are able to support more than one version of TypeScript.
```

Reply:
ahh nvm its difference between British and us

Reply:
Yeah, I'm British. this is a big effort for me :D

Reply:
With what appears to be at least two Brits and an Aussie working on this then maybe that’s enough to actually spell things properly :)